.github/workflows/publish-artifacts.yml

```diff
@@ -53,3 +53,31 @@ jobs:
 
       - name: Publish to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
+
+
+  helm:
+    runs-on: ubuntu-latest
+    needs: docker-images
+    steps:
+      - uses: actions/checkout@v4
+       # v4.3.1
+      - name: Set up Helm
+        uses: azure/setup-helm@1a275c3b69536ee54be43f2070a358922e12c8d4
+        with:
+          version: v3.14.4
+       # v3
+      - name: Log in to Docker Hub
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d
+        with:
+          username: ${{ secrets.DOCKERHUB_RW_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_RW_TOKEN }}
+
+      - name: Package Helm chart
+        working-directory: helm-chart
+        run: |
+          helm dependency update && helm package . --version ${{ github.event.release.tag_name }} && mv cvat-*.tgz ../
```

Comment:
The first command can be replaced with the `--dependency-update` option of `helm package`, and the last command can be replaced with the `--destination` option.